pep-0692.rst

```diff
@@ -376,32 +374,15 @@ to perform the function call::
 
 Intended Usage
 --------------
-
-This proposal will bring a large benefit to the codebases that already use
-``**kwargs`` because of the flexibility that they provided in the initial
-phases of the development, but now are mature enough to use a stricter
-contract via type hints. 
-
-Adding type hints directly in the source code as opposed to the ``*.pyi``
-stubs benefits anyone who reads the code as it is easier to understand. Given
-that currently precise ``**kwargs`` type hinting is impossible in that case the
-choices are to either not type hint ``**kwargs`` at all, which isn't ideal, or
-to refactor the function to use explicit keyword arguments, which often exceeds
-the scope of time and effort allocated to adding type hinting and, as any code
-change, introduces risk for both project maintainers and users. In that case
-hinting ``**kwargs`` using a ``TypedDict`` as described in this PEP will not
-require refactoring and function body and function invocations could be
-appropriately type checked.
-
-Moreover, using ``**kwargs`` with more precise type hinting would allow for
-less code and less copy pasting in cases when a module contains a bunch of
-functions that expect similar keyword arguments. Using ``**kwargs`` with a
-``TypedDict`` as propsed in this PEP is much more concise than explicitly
-repeating the keyword arguments every time. 
-
-Another useful pattern that justifies using and typing ``**kwargs`` as proposed
-is when the function's API should allow for optional keyword arguments that
-don't have default values.
+The intended use cases for this proposal are described in the :ref:`Motivation
+<pep-692-motivation>` section. In summary, more precise ``**kwargs`` typing
```

Comment:
FYI, you can just do

```suggestion
The intended use cases for this proposal are described in the
:ref:`pep-692-motivation` section. In summary, more precise ``**kwargs`` typing
```

and the section name will be included as the reference text.